src/engine/graph_node.c

```diff
@@ -430,7 +430,7 @@ struct GraphNodeShadow *init_graph_node_shadow(struct AllocOnlyPool *pool,
 
     if (graphNode != NULL) {
 #ifdef LEGACY_SHADOW_IDS
-        if (shadowType == 1 || shadowType == 2 || shadowType == 99) {
+        if (shadowType == 0 || shadowType == 1 || shadowType == 2 || shadowType == 99) {
             shadowType = SHADOW_CIRCLE;
```

Comment:
@Arceveti can you add defines for these?
![Screenshot_20220517-183210](https://user-images.githubusercontent.com/79979276/168928715-7f69fe27-ff45-4882-8aae-41b5e1cc7a44.png)

Reply:
Not enums?

Reply:
enums are indeed better

Reply:
done